cli/wallet/nep11.go

```diff
@@ -332,7 +332,7 @@ func printNEP11Owner(ctx *cli.Context, divisible bool) error {
 	}
 
 	if divisible {
-		result, err := c.NEP11DOwnerOf(tokenHash.Uint160(), tokenIDBytes)
+		result, err := c.NEP11DUnpackedOwnerOf(tokenHash.Uint160(), tokenIDBytes)
```

Comment:
Extend documentation to mention that these NEP-11 functions only return some default number of elements?

Reply:
Here I mean CLI documentation, because it affects CLI functions as well.

Reply:
Fixed.